make/config/projectcontour/contour-gateway.yaml

```diff
@@ -5286,3713 +5276,3 @@ spec:
         runAsNonRoot: true
         runAsUser: 65534
         runAsGroup: 65534
-
----
-apiVersion: apiextensions.k8s.io/v1
```

Comment:
I have removed the gateway CRDs as they already get applied once from here https://github.com/cert-manager/cert-manager/blob/master/make/e2e-setup.mk#L211